modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -795,7 +795,9 @@ private Map<String, Object> _toProperties(
 				map.put(objectFieldName, fileEntry);
 			}
 			else if (objectField.compareBusinessType(
-						ObjectFieldConstants.BUSINESS_TYPE_DATE_TIME)) {
+						ObjectFieldConstants.BUSINESS_TYPE_DATE) ||
+					 objectField.compareBusinessType(
+						 ObjectFieldConstants.BUSINESS_TYPE_DATE_TIME)) {
```

Comment:
@marcelabc are we causing a breaking change by changing the headless API return?